video/out/hwdec/hwdec_vaapi_gl.c

```diff
@@ -135,17 +145,25 @@ static void vaapi_gl_mapper_uninit(const struct ra_hwdec_mapper *mapper)
                         p_mapper->desc.layers[n].offset[plane]); \
             ADD_ATTRIB(EGL_DMA_BUF_PLANE ## plane ## _PITCH_EXT, \
                         p_mapper->desc.layers[n].pitch[plane]); \
+            if (p_owner->use_modifiers && \
+                p_mapper->desc.objects[p_mapper->desc.layers[n].object_index[plane]].drm_format_modifier != DRM_FORMAT_MOD_INVALID) { \
+                ADD_ATTRIB(EGL_DMA_BUF_PLANE ## plane ## _MODIFIER_LO_EXT, \
+                           p_mapper->desc.objects[p_mapper->desc.layers[n].object_index[plane]].drm_format_modifier & 0xfffffffful); \
+                ADD_ATTRIB(EGL_DMA_BUF_PLANE ## plane ## _MODIFIER_HI_EXT, \
+                           p_mapper->desc.objects[p_mapper->desc.layers[n].object_index[plane]].drm_format_modifier >> 32); \
```

Comment:
I think it would be overall more readable to just give the modifier in a name (it's inside `do` block so it won't leak outside the scope anyway). Would make the code prettier for sure. 🤷🏻‍♀️

Reply:
done.